lucetc/src/lib.rs

```diff
@@ -98,6 +99,8 @@ pub trait LucetcOpts {
     fn with_verify(self) -> Self;
     fn sign(&mut self);
     fn with_sign(self) -> Self;
+    fn count_instructions(&mut self);
```

Comment:
These should take a `bool` (and now that I'm looking at it, the sign and verify methods should too)